proto/server.proto

```diff
@@ -99,6 +99,44 @@ message RealTimeResponse {
   string error = 4;
   repeated FeedData feedData = 5;
 }
+
+message UserManMeta {
```

Comment:
Just keep use the Meta object? `col` will be unused.

Reply:
Ok